devices/common_patches/h7_exti_dualcore.yaml

```diff
@@ -0,0 +1,853 @@
+          bitOffset: 31
+          bitWidth: 1
+      C1PR1:
+      description: EXTI pending register
+      addressOffset: 0x00C8
+      resetValue: 0x00000000
+      fields:
+        PR0:
+          description: CPU2 event mask on Event input x
+          bitOffset: 00
+          bitWidth: 1
+        PR1:
+          description: CPU2 event mask on Event input x
+          bitOffset: 01
+          bitWidth: 1
+        PR2:
+          description: CPU2 event mask on Event input x
+          bitOffset: 02
+          bitWidth: 1
+        PR3:
+          description: CPU2 event mask on Event input x
+          bitOffset: 03
+          bitWidth: 1
+        PR4:
+          description: CPU2 event mask on Event input x
+          bitOffset: 04
+          bitWidth: 1
+        PR5:
+          description: CPU2 event mask on Event input x
+          bitOffset: 05
+          bitWidth: 1
+        PR6:
+          description: CPU2 event mask on Event input x
+          bitOffset: 06
+          bitWidth: 1
+        PR7:
+          description: CPU2 event mask on Event input x
+          bitOffset: 07
+          bitWidth: 1
+        PR8:
+          description: CPU2 event mask on Event input x
+          bitOffset: 08
+          bitWidth: 1
+        PR9:
+          description: CPU2 event mask on Event input x
+          bitOffset: 09
+          bitWidth: 1
+        PR10:
+          description: CPU2 event mask on Event input x
+          bitOffset: 10
+          bitWidth: 1
+        PR11:
+          description: CPU2 event mask on Event input x
+          bitOffset: 11
+          bitWidth: 1
+        PR12:
+          description: CPU2 event mask on Event input x
+          bitOffset: 12
+          bitWidth: 1
+        PR13:
+          description: CPU2 event mask on Event input x
+          bitOffset: 13
+          bitWidth: 1
+        PR14:
+          description: CPU2 event mask on Event input x
+          bitOffset: 14
+          bitWidth: 1
+        PR15:
+          description: CPU2 event mask on Event input x
+          bitOffset: 15
+          bitWidth: 1
+        PR16:
+          description: CPU2 event mask on Event input x
+          bitOffset: 16
+          bitWidth: 1
+        PR17:
+          description: CPU2 event mask on Event input x
+          bitOffset: 17
+          bitWidth: 1
+        PR18:
+          description: CPU2 event mask on Event input x
+          bitOffset: 18
+          bitWidth: 1
+        PR19:
+          description: CPU2 event mask on Event input x
+          bitOffset: 19
+          bitWidth: 1
+        PR20:
+          description: CPU2 event mask on Event input x
+          bitOffset: 20
+          bitWidth: 1
+        PR21:
+          description: CPU2 event mask on Event input x
+          bitOffset: 21
+          bitWidth: 1
+    C2IMR2:
+      description: EXTI interrupt mask register
+      addressOffset: 0x00D0
+      resetValue: 0xFFF5FFFF
+      fields:
+        MR0:
```

Comment:
RM calls this `MR32` instead of `MR0`, etc for the rest

Reply:
Resolved, only made the mistake here for C2xxx or C3xxx I believe